programmers-coding-test/kimjiwon/binary-search/부품찾기.js

```diff
@@ -0,0 +1,12 @@
+const line = require("fs").readFileSync("/dev/stdin", "utf8");
+let input = line.trim().split("\n");
+const result = [];
+const N = +input[0];
+const product = input[1].split(" ").map(Number);
+const M = +input[2];
+const needed = input[3].split(" ").map(Number);
```

Comment:
개인적인 코테에서는 빠르고 편리해서 '+' 로 정수형 변환은 좋다고 봄.
다만, 협업같이 함께 공유하는 코드에서 이러한 코드를 보았을 때 몇몇은 이해할 수 없을 것이라 판단이 됨.
따라서 개발할 때는 정수형 변환을 +가 아닌 Number 혹은 parseInt로 부탁합니다~

```suggestion
const N = Number(input[0]);
const product = input[1].split(" ").map(Number);
const M = Number(input[2]);
const needed = input[3].split(" ").map(Number);
```